### ISSUE TITLE
Outlook Marketplace

### ISSUE THREAD

**Issue by FrancescoGd** [mmanela/MarkdownOutlook#30]
Would it be possible to have it install through the Marketplace instead of the .msi installer? It'd be useful if (like me) you work in an environment where policies don't allow running an installer without admin privileges but you can install extensions from the Marketplace

**Comment by jvivian-atreca** [comment]
Would also allow for installation on outlook for mac if I'm not mistaken.

**Comment by atanasj** [comment]
Yes, I cannot get this installed via the `.msi` installer on mac.
